changes files:
--------------
--- java-1_8_0-openjdk.changes
+++ java-1_8_0-openjdk.changes
@@ -1,0 +2,138 @@
+Tue Jun  9 13:45:17 UTC 2026 - Fridrich Strba <fstrba@suse.com>
+
+- Require at least gcc 7 on aarch64 since with gcc < 5 is considered
+  broken by upstream on this architecture: "checking for broken
+  aarch64 gcc 4.x... configure: error: GCC < 5 may incorrectly
+  compile HotSpot on aarch64. See JDK-8360869."
+
+-------------------------------------------------------------------
+Thu Jun  4 07:20:26 UTC 2026 - Fridrich Strba <fstrba@suse.com>
+
+- Make post scripts less noisy (bsc#1267355)
+
+-------------------------------------------------------------------
+Fri May 29 07:15:04 UTC 2026 - Fridrich Strba <fstrba@suse.com>
+
+- Use libalternatives instead of update-alternatives for
+  distributions where libalternatives is available

old:
----
  fix-build-with-gcc14.patch
  icedtea-3.38.0.tar.xz

new:
----
  icedtea-3.39.0.tar.xz

spec files:
-----------
--- java-1_8_0-openjdk.spec
+++ java-1_8_0-openjdk.spec
@@ -16,10 +16,16 @@
 #
 
 
+# ugly hack to prevent spec-cleaner from changing make -> make_build
+%global make make
 %{!?aarch64:%global aarch64 aarch64 arm64 armv8}
 %global jit_arches %{ix86} x86_64 ppc64 ppc64le %{aarch64} %{arm}
-%global icedtea_version 3.38.0
+%global icedtea_version 3.39.0
 %global buildoutputdir openjdk.build/
-Patch13:        fix-build-with-gcc14.patch
+%global headless_binaries java keytool orbd policytool rmid rmiregistry servertool tnameserv
+%global headless_binaries_comma %(echo %{headless_binaries} | sed 's#\ #,#g')
+%global devel_binaries javac appletviewer extcheck jar jarsigner javadoc javah javap jcmd jconsole jhat jinfo jmap jps jrunscript jsadebugd jstack jstat jstatd native2ascii rmic schemagen serialver wsgen wsimport xjc
+%global devel_binaries_comma %(echo %{devel_binaries} | sed 's#\ #,#g')
 # Convert an absolute path to a relative path.  Each symbolic link is
 # specified relative to the directory in which it is installed so that
 # it will resolve properly within chrooted installations.
@@ -27,12 +33,18 @@
 %global abs2rel perl -e %{script}
 %global syslibdir       %{_libdir}
 %global archname        %{name}
+%if 0%{?suse_version} > 1500

other changes:
--------------
++++++ aarch32-git.tar.xz
--- make/aix/makefiles/adlc.make
+++ make/aix/makefiles/adlc.make
@@ -70,8 +70,8 @@

new:
----
  _patchinfo

other changes:
--------------
++++++ _patchinfo (new)
--- _patchinfo
+++ _patchinfo
@@ -0,0 +1,147 @@

new:
----
  _patchinfo

other changes:
--------------
++++++ _patchinfo (new)
--- _patchinfo
+++ _patchinfo
@@ -0,0 +1,147 @@

new:
----
  _patchinfo

other changes:
--------------
++++++ _patchinfo (new)
--- _patchinfo
+++ _patchinfo
@@ -0,0 +1,147 @@

new:
----
  _patchinfo

other changes:
--------------
++++++ _patchinfo (new)
--- _patchinfo
+++ _patchinfo
@@ -0,0 +1,147 @@

new:
----
  _patchinfo

other changes:
--------------
++++++ _patchinfo (new)
--- _patchinfo
+++ _patchinfo
@@ -0,0 +1,147 @@

new:
----
  _patchinfo

other changes:
--------------
++++++ _patchinfo (new)
--- _patchinfo
+++ _patchinfo
@@ -0,0 +1,147 @@

new:
----
  _patchinfo

other changes:
--------------
++++++ _patchinfo (new)
--- _patchinfo
+++ _patchinfo
@@ -0,0 +1,147 @@

